core/src/main/java/io/grpc/internal/RetriableStream.java

```diff
@@ -269,7 +291,35 @@ public final void start(ClientStreamListener listener) {
     class StartEntry implements BufferEntry {
       @Override
       public void runWith(Substream substream) {
+        if (hedgingPolicy == null) {
```

Comment:
But `drain()` runs now, as well. So I don't think this is doing anything.

Reply:
Yes, for delayed stream is really difficult to schedule the hedging, in this case need let `drain()` happen, and delay scheduling the hedging once name resolution completes. Otherwise there is no hedging policy, so no hedging at all (Unless there is a local service config). It's too difficult, so I put a comment there.